### PR TITLE
some tags did not have closing tags and indentation was not proper in this file

### PR DESCRIPTION
**Pull request by aashimgarg**
Issue solved #2798

Fixes HTML validation issues in
openlibrary/admin/templates/admin/index.html

Issues:
1. In this index.html file the table data , strong , and br did not have closing tags.
2. the indentation was not proper.

What is done:
1.Made the indentation proper. (as now it is much easier to understand)
2. gave the closing tags.

Stakeholders:
@koderjoker

### CODE REVIEW

**Comment by aashimgarg** [comment]
@jdlrobson @koderjoker

File: openlibrary/admin/templates/admin/index.html [hunk 1]
```diff
@@ -178,12 +186,12 @@ <h2>Members</h2>
         <div class="titleSmall fixthis">Prolific Member Editors</div>
         <div class="smaller">
             <a href="[USER PROFILE]">name</a>,
-            <a href="[USER PROFILE"]>name</a>,
-            <a href="[USER PROFILE"]>name</a>,
-            <a href="[USER PROFILE"]>name</a>,
-            <a href="[USER PROFILE"]>ip address</a>,
-            <a href="[USER PROFILE"]>name</a>,
-            <a href="[USER PROFILE"]>name</a>
+            <a href="[USER PROFILE" ]>name</a>,
```
Comment (jdlrobson) [inline openlibrary/admin/templates/admin/index.html]: Looks like you added some additional spaces here

Comment (jdlrobson) [reply openlibrary/admin/templates/admin/index.html]: And changed how this functions?

Is it
href="[USER PROFILE]"
Or
href="[USER PROFILE"] ?

I'm guessing all of these should be consistent.

**Comment by aashimgarg** [comment]
@jdlrobson is it okay?

**Comment by jdlrobson** [comment]
I was hoping you could reassure me. Haven't looked at these pages before. :)

**Comment by aashimgarg** [comment]
These are the web.py templates for the frontend.

**Comment by jdlrobson** [comment]
I understand what you are doing. 

What I don't understand is what the content after "Prolific Member Editors" does and why you need to change the markup there.

Are you able to get this to display locally?
If so can you post a screenshot of how this renders before and after?

**Comment by aashimgarg** [comment]
I changed the markup because as href attribute of the anchor tag in html <a  href="_"> encloses the link in the double codes.

**Comment by jdlrobson** [comment]
Please post a screenshot.

**Comment by hornc** [comment]
Thanks @aashimgarg

**Comment by hornc** [comment]
@aashimgarg  @jdlrobson I chose to merge this without the screenshots as this is dead-code (unfortunately) and I don't believe you could view this page properly with the current site config.

@jdlrobson was right to ask for screenshots, testing page changes in your local env is always the right thing to do, and we all would have noticed sooner that this page isn't used and can't be viewed :)

I will be adding an issue to investigate some of these unused admin templates so they can be removed.

**Pull request merged**